src/test/unittest/ut_setup.c

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * ut_setup.h -- utility helper functions for libpmem2 tests setup
+ */
+#include "../../libpmem2/config.h"
+#include "ut_fh.h"
+#include "ut_pmem2.h"
+#include "ut_setup.h"
+#include "unittest.h"
+
+/*
+ * ut_prepare_config -- fill pmem2_config, this function can not set
+ * the wrong value
+ */
+void
+ut_prepare_config_internal(struct pmem2_config *cfg, struct pmem2_source **src,
+	struct FHandle **fh, enum file_handle_type fh_type, const char *file,
```

Comment:
This intends to simplify the initialization of files, but it takes three different parameters - I'm not sure how to fix this, but it just seems odd.